manifests/config.pp

```diff
@@ -41,7 +41,7 @@
   foreman_proxy::settings_file { 'tftp': }
   foreman_proxy::settings_file { 'realm': }
 
-  if $foreman_proxy::puppetca or $foreman_proxy::puppetrun {
+  if $foreman_proxy::puppetca or $foreman_proxy::puppetrun or tagged('foreman_proxy::plugin::salt') {
```

Comment:
I'm not sure what would be the fix here. Maybe the sudo config needs to become pluggable through fragments?